internal/ihop/user_layer_creator.go

```diff
@@ -87,7 +87,7 @@ func (c UserLayerCreator) Create(image Image, def DefinitionImage, _ SBOM) (Laye
 	err = tw.WriteHeader(&tar.Header{
 		Typeflag: tar.TypeDir,
 		Name:     "home/cnb",
-		Mode:     int64(os.ModePerm),
+		Mode:     int64(os.FileMode(750)),
```

Comment:
Hey @candrews thanks for the PR! Are you able to address the linting error that's showing up on this line?

Reply:
I've pushed an updated to this PR fixing the linting error, thank you for pointing it out.